crates/red_knot_python_semantic/src/types/signatures.rs

```diff
@@ -92,6 +101,23 @@ impl<'db> Parameters<'db> {
         ])
     }
 
+    /// Return parameters that accepts any arguments i.e., `(*args: Any, **kwargs: Any)`
+    pub(crate) fn any() -> Self {
+        // TODO: Should this be represented using `(*Any, **Any)` instead?
```

Comment:
Oh, yeah, maybe we should ideally omit the names here? Not sure how much fallout that change would have

Reply:
Now that we've decided to use `(...) -> Unknown` I don't think this is relevant so I've avoided this.